### PR TITLE
Add personal accounts

### PR DESCRIPTION
**Pull request by antlu**
Changes needed for https://github.com/UffizziCloud/uffizzi_platform/pull/238.

### CODE REVIEW

File: core/app/controllers/uffizzi_core/api/cli/v1/accounts/projects_controller.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class UffizziCore::Api::Cli::V1::Accounts::ProjectsController < UffizziCore::Api
 
   def create
     project_form = UffizziCore::Api::Cli::V1::Project::CreateForm.new(project_params)
-    project_form.account = current_user.organizational_account
+    project_form.account = current_user.personal_account
```
Comment (moklidia) [inline core/app/controllers/uffizzi_core/api/cli/v1/accounts/projects_controller.rb:20]: Let's move the `default_account` definition to the app and use it in both core and the platform

File: core/app/controllers/uffizzi_core/api/cli/v1/accounts/projects_controller.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ class UffizziCore::Api::Cli::V1::Accounts::ProjectsController < UffizziCore::Api
 
   def create
     project_form = UffizziCore::Api::Cli::V1::Project::CreateForm.new(project_params)
-    project_form.account = current_user.organizational_account
+    project_form.account = current_user.default_account
```
Comment (moklidia) [inline core/app/controllers/uffizzi_core/api/cli/v1/accounts/projects_controller.rb:20]: We need to find the account here by the `account_id` passed in query params

File: core/app/services/uffizzi_core/project_service.rb [hunk 3]
```diff
@@ -39,7 +39,7 @@ def update_compose_secret_errors(project, secret)
     def add_users_to_project!(project, current_user)
       user_projects = []
 
-      current_user.organizational_account.memberships.where(role: UffizziCore::Membership.role.admin).map do |membership|
+      current_user.default_account.memberships.where(role: UffizziCore::Membership.role.admin).map do |membership|
```
Comment (moklidia) [inline core/app/services/uffizzi_core/project_service.rb:42]: Need to pass account to this method as an argument

File: core/app/services/uffizzi_core/project_service.rb [hunk 4]
```diff
@@ -36,14 +36,12 @@ def update_compose_secret_errors(project, secret)
...
 
-    def add_users_to_project!(project, current_user)
-      user_projects = []
+    def add_user_to_project(user, project, account)
+      is_account_owner = account.owner_id == user.id
+      role = is_account_owner ? UffizziCore::UserProject.role.admin : UffizziCore::UserProject.role.developer
 
-      current_user.default_account.memberships.where(role: UffizziCore::Membership.role.admin).map do |membership|
-        user_projects << { project: project, user: membership.user, role: UffizziCore::UserProject.role.admin }
-      end
```
Comment (moklidia) [inline core/app/services/uffizzi_core/project_service.rb:44]: When we create a new project, we need to add all account admins to the project as admins, this is what this code is for

**Review by zipofar — APPROVED** [review]

**Pull request merged**